agent/grpc-internal/client.go

```diff
@@ -132,8 +133,6 @@ func (c *ClientConnPool) dial(datacenter string, serverType string) (*grpc.Clien
 		grpc.WithContextDialer(c.dialer),
 		grpc.WithDisableRetry(),
 		grpc.WithStatsHandler(agentmiddleware.NewStatsHandler(metrics.Default(), metricsLabels)),
-		// nolint:staticcheck // there is no other supported alternative to WithBalancerName
-		grpc.WithBalancerName("pick_first"),
```

Comment:
How are you going to turn on the "pick the first server every time" behavior now?

Reply:
I believe "pick_first" is the default balancer.

If I'm wrong, then this should be the new way to define balancer config: https://github.com/weaveworks/common/issues/239#issuecomment-1115920318